.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.
+
+## Motivation
+
+Please describe the motivation of this PR and the goal you want to achieve through this PR.
+
+## Modification
+
+Please briefly describe what modification is made in this PR.
+
+## BC-breaking (Optional)
+
+Does the modification introduce changes that break the backward compatibility of the downstream repos?
+If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.
+
+## Use cases (Optional)
+
+If this PR introduces a new feature, it is better to list some use cases here and update the documentation.
+
+## Checklist
+
+- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
+- [ ] The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
+- [ ] If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMSeg.
+- [ ] The documentation has been modified accordingly, like docstring or example tutorials.
```

Comment:
Do we need to attach some corresponding tutorial links here? Otherwise, they may continue to ask how to pass pre-commit and how to test. . .

Reply:
I think we have put the pre-commit hook guide in [CONTRIBUTING.md](https://github.com/open-mmlab/mmclassification/blob/master/.github/CONTRIBUTING.md). And we should assume contributers have read it, here is just a reminding.